### PR TITLE
Antimeta box set - ops

### PR DESCRIPTION
**Pull request by yackemflam**
:cl: yackemflam
add: OOOOPS
/:cl:

### CODE REVIEW

**Comment by nullbear** [comment]
Actually i think this is great.

File: code/game/objects/items/weapons/storage/uplink_kits.dm [hunk 1]
```diff
@@ -112,14 +112,25 @@
...
+		if("sniper") //This shit is unique so can't really balance it around tc, also no silencer because getting killed without ANY indicator on what killed you sucks
 			new /obj/item/weapon/gun/projectile/automatic/sniper_rifle(src)
 			new /obj/item/ammo_box/magazine/sniper_rounds/penetrator(src)
 			new /obj/item/clothing/glasses/thermal/syndi(src)
 			new /obj/item/clothing/gloves/color/latex/nitrile(src)
 			new /obj/item/clothing/mask/gas/clown_hat(src)
 			new /obj/item/clothing/under/suit_jacket/really_black(src)
 			return
+		
+		if("metaops") // 30 tc / Meta breaking is going to be my new job - Yackemflam
```
Comment (JJRcop) [inline code/game/objects/items/weapons/storage/uplink_kits.dm]: Something something author name in comments

**Comment by AdamElTablawy** [comment]
>a fucking bulldog shotgun for traitors

File: code/game/objects/items/weapons/storage/uplink_kits.dm [hunk 2]
```diff
@@ -112,14 +112,25 @@
...
+			new /obj/item/clothing/head/helmet/space/hardsuit/syndi(src) // 8tc
```
Comment (PKPenguin321) [inline code/game/objects/items/weapons/storage/uplink_kits.dm]: You're not supposed to be able to get hardsuit helmets as an individual item

**Comment by ma44** [comment]
You to can become a nuke ops as well!

**Comment by Bawhoppen** [comment]
this is dumb because of reusing weapons which is uncreative

**Comment by nullbear** [comment]
So weapons should only be used once and forgotten? I think this has a really good reason for their reuse. Ie. It being the same faction lore wise.  The guns only having one way to get them currently. And anti-meta.

**Comment by KorPhaeron** [comment]
Metabreaking is a meme until datum antags. Only new players fall for it and the scare lasts about five minutes.

**Comment by nullbear** [comment]
Datum antags. Meta antags that exist in the code and randomly attack different game mechanics, causing file corruption until the next compile.

**Comment by yackemflam** [comment]
Not, unless someone actually FINISHES datum antag this is the next best thing.
And no, people have been tricked before easily.

**Comment by Armhulen** [comment]
wait, bulldog shotgun is 8 tc?

????

**Pull request closed without merging**